examples/vault-auto-unseal/README.md

```diff
@@ -0,0 +1,56 @@
+# Vault auto unseal example
+
+** For Vault Enterprise version only **
+
+This folder shows an example of Terraform code to deploy a [Vault][vault] cluster in
+Amazon AWS using the [vault-cluster module][vault_cluster] and use the Vault Enterprise
+feature of auto unsealing the cluster through Amazon KMS.
```

Comment:
(instead of manually unsealing by having multiple admins enter their unseal keys by hand)